compiler/codegen/library.cpp

```diff
@@ -813,7 +811,7 @@ void codegen_make_python_module() {
 
 // Skip this function if it is defined in an internal module, or if it is
 // the generated main function
-static bool isFunctionToSkip(FnSymbol* fn) {
+bool isInternalExport(FnSymbol* fn) {
```

Comment:
How about something like `isUserRoutine` instead, and then just using the negation of that? The problem I have with "isInternal" is that:

- When I read it I think it's referring to internal modules (as opposed to standard modules)
- The method is checking to see if a routine is from a standard or internal module, nothing to do with exports